docs/core/porting/index.md

```diff
@@ -13,35 +13,68 @@ You might have code that currently runs on the .NET Framework that you're intere
 
 ## Overview of the porting process
 
-We recommend you use the following process when porting your project to .NET Core:
+We recommend you use the following process when porting your project to .NET Core. Each of these steps introduce potential places for behavior changes, so ensure that you adequately test your library or application before continuing on to later steps. The first steps are to get your project ready for a switch to .NET Standard/.NET Core. If you have unit tests, it is best to convert them first so that you can continue testing changes in the product you are working on. Because porting to .NET Core is such a significant change to your codebase, it's highly recommended to port your test projects so that you can run tests as you port your code over. MSTest, xUnit, and NUnit all work on .NET Core.
```

Comment:
```suggestion
We recommend you use the following process when porting your project to .NET Core. Each of these steps introduces potential places for behavior changes, so ensure that you adequately test your library or application before continuing on to later steps. The first steps are to get your project ready for a switch to .NET Standard or .NET Core. If you have unit tests, it's best to convert them first so that you can continue testing changes in the product you're working on. Because porting to .NET Core is such a significant change to your codebase, it's highly recommended to port your test projects so that you can run tests as you port your code over. MSTest, xUnit, and NUnit all work on .NET Core.
```

Reply:
Done